build/configs/rtl8730e/loadable_ext_psram/defconfig

```diff
@@ -559,8 +559,9 @@ CONFIG_AUDIO_KEYWORD_DETECT=y
 # CONFIG_AUDIO_ALC5658 is not set
 # CONFIG_AUDIO_NULL is not set
 # CONFIG_AUDIO_CX20921 is not set
-CONFIG_AUDIO_ALC1019=y
+# CONFIG_AUDIO_ALC1019=y
```

Comment:
You modify the file manually. It should not be. Please modify this through the menuconfig and then copy .config file into this.